java/test/jmri/jmrit/display/logixng/configurexml/StoreAndLoadTest.java

```diff
@@ -303,9 +303,8 @@ public void setUp() {
 //        JUnitUtil.initLogixNGManager();
     }
 
-    @After
+    @AfterEach
     public void tearDown() {
-        // JUnitAppender.clearBacklog();    // REMOVE THIS!!!
```

Comment:
I would like this comment to stay. There is an important difference between JUnit4 and JUnit5. In JUnit4, if two errors occurred, both was shown. But in JUnit5, only a summary is shown. For me, this is a big problem, making it much harder to track down the problems.

The issue here is that almost always when an error occurrs, the error is logged. And the JMRI test suite checks that the log is empty, otherwise the test fails. This means that if one error occur, you will get two errors. The original error and the message in the log.

To find the original error, I force empty of the log if I see an error. That's done by `clearBacklog()`. But that statement must not be left uncommented, since otherwise it might hide other errors. Therefore I have the comment `// REMOVE THIS!!!`.

Reply:
Hi @danielb987 , no problem, have restored the lines,
Steve.

Reply:
Thanks!